packages/jest-emotion/serializer.js

```diff
@@ -0,0 +1,4 @@
+const { createSerializer } = require('./dist')
```

Comment:
Could you change this to requiring `jest-emotion` instead of `./dist`?

Reply:
Interesting! Will that work!?

Reply:
Yeah, node should look in the node_modules folder, go into the jest-emotion folder and look at the main property in the package.json.